src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/create_index_pattern_wizard.tsx

```diff
@@ -352,6 +353,7 @@ export class CreateIndexPatternWizard extends Component<
 
     return (
       <>
+        <ExperimentalCallout />
```

Comment:
could you add this behind this switch `dataSourceEnabled`. Only display the banner when true

Reply:
@yibow98  https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/index_pattern_management/public/plugin.ts#L51 Start from here~

Reply:
@noCharger Thanks! Done.